fe/fe-core/src/main/java/org/apache/doris/nereids/stats/FilterSelectivityCalculator.java

```diff
@@ -107,4 +114,9 @@ public Double visitLessThan(LessThan lessThan, Void context) {
         return DEFAULT_SELECTIVITY;
     }
 
+    @Override
+    public Double visitInPredicate(InPredicate inPredicate, Void context) {
```

Comment:
maybe we need to let this class inherit ExpressionVisitor and then Override visit function like what u do, and remove all visitXXX function that do same thing with visit function